nuklear.h

```diff
@@ -15838,13 +15838,12 @@ nk_draw_checkbox(struct nk_command_buffer *out,
 
     /* draw background and cursor */
     if (background->type == NK_STYLE_ITEM_COLOR) {
-        nk_fill_rect(out, *selector, 0, style->border_color);
-        nk_fill_rect(out, nk_shrink_rect(*selector, style->border), 0, background->data.color);
+		nk_stroke_rect(out, *selector, 2, 2, background->data.color);
```

Comment:
It's a total nitpicking, but this indentation is not correct :wink:.

Reply:
I have looked at the codes ,It are correct!

Reply:
The picture is totally fine. It's just, that this line is indented with tabs, but shall be indented with spaces. Otherwise I think this commit can get merged.